src/crawler.ts

```diff
@@ -2462,6 +2555,18 @@ self.__bx_behaviors.selectMainBehavior();
       return false;
     }
 
+    if (this.params.robots) {
+      const robots = await this.fetchAndParseRobots(url, logDetails);
+      if (robots && robots.isDisallowed(url, "Browsertrix/1.0")) {
```

Comment:
Should we make the robots useragent configurable (separate from crawling UA), eg. --robotsUA ?

Reply:
I suppose it can default to `Browsertrix/<current package>` version

Reply:
So can make it configurable, but I guess the question first is: do we actually want to send the request for robots.txt with this different user agent? Or are we just trying to use it in parsing the response (being a small enough fish for now that we're unlikely to ever come across any specific rules for Browsertrix)?